components/climber.py

```diff
@@ -1,13 +1,45 @@
+import wpilib
+from rev import CANSparkMax
+from ids import SparkMaxIds
+
+
 class ClimberComponent:
     def __init__(self) -> None:
         self.deployed = False
 
     def deploy(self) -> None:
         self.deployed = True
 
+    def execute(self) -> None:
+        if self.deployed:
+
+            def __init__(self) -> None:
+                self.deployed = False
+
+    def deploy(self) -> None:
+        self.deployed = True
+
     def execute(self) -> None:
         if self.deployed:
             # Deploy the climber
-            pass
+            self.climbingmotor = CANSparkMax(
+                SparkMaxIds.climber, CANSparkMax.MotorType.kBrushless
+            )  # using a Neo with 4:1 4:1 3:1 ratio
+            self.limitswitch = wpilib.LimitSwitch(2)
+
+            def motorCurrent():
```

Comment:
I think this class method is unnecessary as you can always just call the method for output current with your motor straight away. Packaging this into a function seems unnecessary